package.json

```diff
@@ -6,19 +6,13 @@
     "start": "react-native start",
     "start:logger": "LOGGER_ENABLED=true react-native start --reset-cache",
     "start:tron": "TRON_ENABLED=true react-native start --reset-cache",
-    "start:android":
-      "concurrently -r 'react-native start' 'yarn start:android:no-packager'",
-    "start:android:logger":
-      "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'yarn start:android:no-packager'",
-    "start:android:tron":
-      "TRON_ENABLED=true concurrently -r 'react-native start --reset-cache' 'yarn start:android:no-packager'",
+    "start:android": "concurrently -r 'react-native start' 'yarn start:android:no-packager'",
+    "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'yarn start:android:no-packager'",
+    "start:android:tron": "TRON_ENABLED=true concurrently -r 'react-native start --reset-cache' 'yarn start:android:no-packager'",
```

Comment:
I don't know why those lines are formatted in #706. But glad to see they are recovered.

Reply:
I might have opened `package.json` and vscode and it got formatted 🤔